store/modules/actions/ui.js

```diff
@@ -17,3 +19,12 @@ export const setErrorMessage = ({ commit }, message) => {
 export const closeErrorDialog = ({ commit }) => {
   commit(types.UI_CLOSE_ERROR_DIALOG);
 };
+
+export async function setLocale({ commit }, locale) {
+  await this.app.i18n.setLocaleAsync(locale);
+  VueCookie.set('language', locale, { expires: '1M' });
```

Comment:
With reference to https://github.com/likecoin/puttyimages-web/pull/124/files#diff-1087e23cd68c057679a0ece386cfd7b0R67

Seem 1 month /30d worth a config with default to 30 day.